prod-sample/sample-modules/late-frank/0.0.1/src/index.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import ModuleContainer from './components/LateFrank';
+
+if (!global.BROWSER) {
+  ModuleContainer.appConfig = {
```

Comment:
this should be done by the bundler

Reply:
What do you mean by this specifically? The configuration?

Reply:
I see what you mean now, will update.

Reply:
updated aa0b3a1